app/App/Panel/index.js

```diff
@@ -47,6 +53,15 @@ class Panel extends React.Component {
   //   })
   // }
 
+  // glitch (el) {
```

Comment:
I think all these comments were moved over to another component, can we get rid of them?

Reply:
y